src/EnergyPlus/Plant/EquipAndOperations.cc

```diff
@@ -993,6 +993,28 @@ namespace DataPlant {
             }
         }
 
+        // override reset AirSourcePlantHeatingOnly to false and AirSourcePlantCoolingOnly to true if the plant cooling load is higher
+        // the plant heating load and the plant heating load is small.
+        if (!this->PlantOps.AirSourcePlantCoolingOnly && this->PlantOps.AirSourcePlantHeatingOnly &&
+            !this->PlantOps.AirSourcePlantSimultaneousHeatingAndCooling) { // all off
+            if (std::abs(this->Report.PrimaryPlantCoolingLoad) > this->Report.PrimaryPlantHeatingLoad &&
+                this->Report.PrimaryPlantHeatingLoad < DataHVACGlobals::SmallLoad) {
+                this->PlantOps.AirSourcePlantCoolingOnly = true;
+                this->PlantOps.AirSourcePlantHeatingOnly = false;
+            }
+        }
```

Comment:
This change is intended to turn on the cooling plant equipment `ON` when the cooling plant loop load is larger than the heating plant loop load, and the plant heating load is very small or `zero`.  This change alters equipment dispatch determined based on the building's aggregate cooling and heating loads only.